Kickstarter-iOS/Views/Controllers/RootTabBarViewController.swift

```diff
@@ -184,11 +184,11 @@ public final class RootTabBarViewController: UITabBarController {
   }
 
   private func viewControllerAndParam<T, P>(with index: RootViewControllerIndex, param: P) -> (T, P)? {
-    guard
-      let vcs = self.viewControllers,
+    guard let vcs = self.viewControllers,
```

Comment:
Can we make this small change for readability and to reuse the existing function to extract the VC:
```swift
guard
  let vcs = self.viewControllers,
  let vc = extractViewController(vcs[clamp(0, vcs.count - 1)(index)]) as? T
else { return nil }
```